### PR TITLE
ocaml-gettext 0.3.6

### PR DESCRIPTION
**Pull request by gildor478**

### CODE REVIEW

**Comment by gildor478** [comment]
@gasche can you have a look and merge?

**Comment by gasche** [comment]
I don't actually have commit rights on the opam-repository, but that seems ok to me. Maybe, given that you are the maintainer, it would make sense to change the `opam` files to follow the "opam lint" recommendations as reported by @camelus? (The only think to know, I think, is that `dev-repo` URLs are expected to end in `.git`)

File: packages/gettext/gettext.0.3.6/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
+opam-version: "1.2"
+maintainer: "Sylvain Le Gall <sylvain+ocaml@le-gall.net>"
+authors: [ "Sylvain Le Gall" ]
+license: "LGPL-2.1 with OCaml linking exception"
+homepage: "https://github.com/gildor478/ocaml-gettext"
+dev-repo: "https://github.com/gildor478/ocaml-gettext.git"
+bug-reports: "https://github.com/gildor478/ocaml-gettext/issues"
+build: [
+  [
+    "./configure"
```
Comment (yallop) [inline packages/gettext/gettext.0.3.6/opam]: I think this is failing because the `configure` script is not included in the tarball.

Comment (gildor478) [inline packages/gettext/gettext.0.3.6/opam]: should be fixed with the new release 0.3.7

**Pull request closed without merging**